src/data-tutorials/users.tsx

```diff
@@ -1244,6 +1244,13 @@ const Users: User[] = [
     website: 'https://learn.figment.io/tutorials/create-vault-smart-contract',
     tags: ['advanced', 'figment', 'react', 'nodejs', 'smartcontract', 'truffle'],
   },
+  {
+    title: 'Introduction to Token Economics on Celo',
+    description: 'A brief overview of the economic principles governing the Celo network and its native token',
+    preview: require('./showcase/intermediate/token.png'),
+    website: 'blog/tutorials/introduction-to-token-economics-on-celo',
```

Comment:
The website and slug has to be exact same, in your case the casing is different , make sure to change that